tensorboard/webapp/routes/dashboard_deeplink_provider.ts

```diff
@@ -106,10 +105,17 @@ export class DashboardDeepLinkProvider extends DeepLinkProvider {
     return combineLatest([
       this.getMetricsPinnedCards(store),
       this.getFeatureFlagStates(store),
-      store.select(getMetricsScalarSmoothing).pipe(
-        map((smoothing) => {
-          if (smoothing === METRICS_SETTINGS_DEFAULT.scalarSmoothing) return [];
-          return [{key: SMOOTHING_KEY, value: String(smoothing)}];
+      store.select(selectors.getMetricsSettingOverrides).pipe(
+        map((settingOverrides) => {
+          if (Number.isFinite(settingOverrides.scalarSmoothing)) {
```

Comment:
While you are correct that I am abusing the `Number.isFinite(undefined)`, I am also using it for `Number.isFinite(NaN) === false)`

Reply:
I see.  Then separately, I think it would be better if metrics_reducer would be responsible for discarding NaNs, so that store-readers would not have to do this isFinite check.  Right now, the settings pane does not do this check when using the selector.